pages/en/lb3/Environment-specific-configuration.md

```diff
@@ -21,6 +21,8 @@ LoopBack applications have the following types of configuration files:
 * **Middleware configuration files**, by default `server/middleware.json`.
 * **Configuration files for LoopBack components**, by default `server/component-config.json`.
 
+{% include note.html content="Using `*.js` files to configure the LoopBack application will only work when the original `.json` files are kept in place. Rather than replacing `.json` files with `.js` files, you should be overriding values from `.json` files in `.js` files. That said, by leaving the default `.json` files empty, all configuration can be done in `.js` files." %}
```

Comment:
Due to a quirk with the "note" Jekyll include, you need to put the ending characters on a separate line, otherwise the note doesn't terminate and consumes the remainder of the page! 
e.g. like this:
```
...can be done in `.js` files.
" %}
```